config/config.txt
{
"title":"iMAPS",
"subtitle":"City of Raleigh & Wake County",
"map":{
"extent":
{
"xmin":2001465.400000,
"ymin":644117.900000,
"xmax":2221323.300000,
"ymax":846796.200000,
"spatialReference":{"wkid":2264}
},
"geometry":"http://maps.raleighnc.gov/arcgis/rest/services/Utilities/Geometry/GeometryServer",
"baselayers":
[
{
"id":"basemap",
"label":"Detailed",
"type":"tiled",
"visible":true,
"opacity":1.00,
"url":"http://maps.raleighnc.gov/ArcGIS/rest/services/BaseMapMobile/MapServer"
},
{
"id":"basemapbasic",
"label":"Basic",
"type":"dynamic",
"visible":false,
"opacity":1.00,
"url":"http://maps.raleighnc.gov/ArcGIS/rest/services/BaseMapBasic/MapServer"
}
],
"imagelayers":
[
{
 "id":"image2014",
 "label":"2014",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos2014/ImageServer",
 "countywide":false
},
{
 "id":"image2013",
 "label":"2013",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos2013/ImageServer",
 "countywide":true
},
{
 "id":"image2012",
 "label":"2012",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos2012/ImageServer",
 "countywide":false
},
{
 "id":"image2011",
 "label":"2011",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos2011/ImageServer",
 "countywide":false
},
{
 "id":"image2010",
 "label":"2010",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos2010/ImageServer",
 "countywide":true
},
{
 "id":"image2009",
 "label":"2009",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos2009/ImageServer",
 "countywide":false
},
{
 "id":"image2008",
 "label":"2008",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos2008/ImageServer",
 "countywide":false
},
{
 "id":"image2007",
 "label":"2007",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos2007/ImageServer",
 "countywide":false
},
{
 "id":"image2006",
 "label":"2006",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos2006/ImageServer",
 "countywide":false
},
{
 "id":"image2005",
 "label":"2005",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos2005/ImageServer",
 "countywide":true
},
{
 "id":"image2004",
 "label":"2004",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos2004/ImageServer",
 "countywide":false
},
{
 "id":"image2002",
 "label":"2002",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos2002/ImageServer",
 "countywide":false
},
{
 "id":"image1999",
 "label":"1999",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos1999/ImageServer",
 "countywide":true
},
{
 "id":"image1988",
 "label":"1988",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos1988/ImageServer",
 "countywide":true
},
{
 "id":"image1981",
 "label":"1981",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos1981/ImageServer",
 "countywide":true
}
],"oplayers":[
 {"id":"corporate",
 "label":"Corporate Boundaries",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Planning/Jurisdictions/MapServer",
 "opacity":0.70,
 "inlist":true},
 {"id":"publicsafety",
 "label":"Public Safety",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/PublicSafety/PublicSafety/MapServer",
 "opacity":0.60,
 "inlist":true}, 
 {"id":"zoning",
 "label":"Zoning",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Planning/Zoning/MapServer",
 "opacity":0.50,
 "visiblelayers":[-1],
 "minscale":76800,
 "inlist":true},  
 {"id":"leafcollection",
 "label":"Leaf Collection Areas",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/PublicWorks/LeafCollection/MapServer",
 "opacity":0.50,
 "inlist":true},
 {"id":"floodplains",
 "label":"Floodplains",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Environmental/Floodplains/MapServer",
 "opacity":0.60,
  "minscale":76800,
 "inlist":true,
 "template":{
  "fieldInfos":[
    {"fieldName":"ZONE_IMAPS",
    "label":"Zone"}
  ]
 }},    
 {"id":"watersheds",
 "label":"Watersheds",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Environmental/Watersheds/MapServer",
 "opacity":0.50,
 "inlist":true},  
 {"id":"swiftcreek",
 "label":"Swift Creek Critical Area",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Environmental/SwiftCreek/MapServer",
 "opacity":0.75,
 "inlist":true},  
 {"id":"riparianbuffers",
 "label":"Riparian Buffers",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Environmental/RiparianBuffers/MapServer",
 "opacity":0.50,
 "minscale":19200,
 "inlist":true},  
 {"id":"parcels",
 "label":"Parcels",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Parcels/MapServer",
 "opacity":1.00,
 "minscale":9600,
 "inlist":true}, 
 {"id":"topography",
 "label":"Topography",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Environmental/Topo/MapServer",
 "opacity":1.00,
 "minscale":4800,
 "inlist":true}, 
 {"id":"greenways",
 "label":"Greenways",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Parks/Greenway/MapServer",
 "opacity":1.00,
 "inlist":true}, 
 {"id":"bluelinestreams",
 "label":"USGS Blueline Streams",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Environmental/USGS_Streams/MapServer",
 "opacity":1.00,
 "inlist":true},    
 {"id":"easements",
 "label":"Easements",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/PublicUtility/Easements/MapServer",
 "opacity":1.00,
 "minscale":19200,
 "inlist":true}, 
 {"id":"sewer",
 "label":"Sewer",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/PublicUtility/SewerExternal/MapServer",
 "opacity":1.00,
 "minscale":19200,
 "inlist":true},  
 {"id":"stormwater",
 "label":"Raleigh Stormwater",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/PublicWorks/Stormwater/MapServer",
 "opacity":1.00,
 "inlist":true},  
  {"id":"streetplan",
 "label":"Raleigh Streetplan",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Planning/StreetPlan/MapServer",
 "opacity":1.00,
 "inlist":true}, 
 {"id":"devplans",
 "label":"Raleigh Development Plans",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Planning/RalDevPlans/MapServer",
 "opacity":1.00,
 "minscale":19200,
 "inlist":true}, 
 {"id":"overlays",
 "label":"Zoning Overlays",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Planning/Overlays/MapServer",
 "opacity":1.00,
 "minscale":76800,
 "inlist":true}, 
 {"id":"subdivisions",
 "label":"Subdivisions",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Planning/Subdivisions/MapServer",
 "opacity":1.00,
 "minscale":19200,
 "inlist":true},    
 {"id":"septic",
 "label":"Septic",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Environmental/SepticTanks/MapServer",
 "opacity":1.00,
 "minscale":38400,
 "inlist":true}, 
 {"id":"wells",
 "label":"Wells",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Environmental/Wells/MapServer",
 "opacity":1.00,
 "minscale":19200,
 "inlist":true},
 {"id":"census",
 "label":"Demographics",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Census/MapServer",
 "opacity":1.00,
 "inlist":true},   
 {"id":"specialuse",
 "label":"Special Zoning Uses",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Planning/SpecialUse/MapServer",
 "opacity":0.60,
 "inlist":true},
 {"id":"boundaries",
 "label":"Boundaries",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Boundaries/MapServer",
 "opacity":0.80,
 "inlist":true},  
 {"id":"urbanprogress",
 "label":"Raleigh Urban Progress Zone",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Planning/UrbanProgressZone/MapServer",
 "opacity":0.50,
 "minscale":76800,
 "inlist":true},                        
 {"id":"urbanform",
 "label":"Raleigh Urban Form",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Planning/UrbanForm/MapServer",
 "opacity":0.80,
 "minscale":100000,
 "inlist":true},
 {"id":"futurelanduse",
 "label":"Raleigh Future Land Use",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Planning/FutureLandUse/MapServer",
 "opacity":0.60,
 "inlist":true},
 {"id":"redevelopment",
 "label":"Raleigh Redevelopment Areas",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Planning/RedevelopmentAreas/MapServer",
 "opacity":1.00,
 "minscale":76800,
 "inlist":true},
{"id":"permits",
 "label":"Raleigh Permit History",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Planning/Permit_History/MapServer",
 "opacity":1.00,
 "minscale":38400,
 "inlist":true},
{"id":"poi",
 "label":"Places of Interest",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/POI1/MapServer",
 "opacity":1.00,
 "minscale":76800,
 "inlist":true} 
],"labels": {"id":"labels",
 "label":"Labels",
 "type":"tiled",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Labels/MapServer",
 "opacity":1.00,
 "inlist":false}
},
"tools":[
 {
  "id":"identify",
  "label":"Identify",
  "icon":"./img/info_default.png",
  "toggle":true,
  "script":"js/plugins/jquery.identify.js",
  "plugin":"identify"  
 },
 {
  "id":"select",  
  "label":"Select",
  "icon":"./img/select_point_default.png",
  "toggle":true,
  "script":"js/plugins/jquery.property-select.js",
  "plugin":"propertySelect"  
 }, 
 {
  "id":"streetview",  
  "label":"Streetview",
  "icon":"./img/streetview_default.png",
  "toggle":true,
  "script":"js/plugins/jquery.streetview.js",
  "plugin":"streetview"  
 },
{
  "id":"measure",  
  "label":"Measure",
  "icon":"./img/measure_default.png",
  "toggle":false,
  "script":"js/plugins/jquery.measure.js",
  "plugin":"measure"
 },
 {
  "id":"bookmarks",  
  "label":"Bookmarks",
  "toggle":false,
  "icon":"./img/bookmark_default.png",
  "script":"js/plugins/jquery.bookmarks.js",
  "plugin":"bookmarks"
 },
 {
  "id":"markup",    
  "icon":"./img/markup_default.png",
  "label":"Markup",
  "toggle":false,
  "script":"js/plugins/jquery.markup-dialog.js",
  "plugin":"markup"
 },
 {
  "id":"print",  
  "label":"Print",
  "icon":"./img/print_default.png",
  "script":"js/plugins/jquery.print-dialog.js",  
  "plugin":"print",
  "toggle":false
 }],
"panel":
{
 "tools":[
  {
  	"label":"Property Search",
    "divid":"propertydiv",
    "script":"js/plugins/jquery.property-search.js",
    "plugin":"propertySearch"
  },
  {
  	"label":"Location Search",
    "divid":"locationdiv",
    "script":"js/plugins/jquery.location-search.js",
    "plugin":"locationSearch"
  },
  {
  	"label":"Layers",
    "divid":"layersdiv",
    "script":"js/plugins/jquery.layer-list.js",
    "plugin":"layerlist" 
  }
 ]
},
"property":
{
  "soe":"http://maps.raleighnc.gov/arcgis/rest/services/Parcels/MapServer/exts/PropertySOE",
  "realEstateOp":"RealEstateSearch",
  "photoOp":"PhotoSearch",
  "deedOp":"DeedSearch",
  "addressOp":"AddressSearch",
  "septicOp":"Septic Permits",
  "propertyService":"http://maps.raleighnc.gov/arcgis/rest/services/Parcels/MapServer/0",
  "symbolMultiple":{
    "type":"esriSFS",
    "style":"esriSFSNull",
    "color":[255,0,0],
    "outline":{
      "type":"esriSLS",
      "style":"esriSLSSolid",
      "color":[255,255,0],
      "width":2
    }
  },
  "symbolSingle":{
    "type":"esriSFS",
    "style":"esriSFSNull",
    "color":[255,0,0],
    "outline":{
      "type":"esriSLS",
      "style":"esriSLSSolid",
      "color":[255,0,0],
      "width":2
    }
  }
},"location":
{
  "geocoder":"http://maps.raleighnc.gov/arcgis/rest/services/Locators/WakeLocator/GeocodeServer",

js/config.js
var config = 
{
"title":"iMAPS",
"subtitle":"City of Raleigh & Wake County",
"map":{
"extent":
{
"xmin":2001465.400000,
"ymin":644117.900000,
"xmax":2221323.300000,
"ymax":846796.200000,
"spatialReference":{"wkid":2264}
},
"geometry":"http://maps.raleighnc.gov/arcgis/rest/services/Utilities/Geometry/GeometryServer",
"baselayers":
[
{
"id":"basemap",
"label":"Detailed",
"type":"tiled",
"visible":true,
"opacity":1.00,
"url":"http://maps.raleighnc.gov/ArcGIS/rest/services/BaseMapMobile/MapServer"
},
{
"id":"basemapbasic",
"label":"Basic",
"type":"dynamic",
"visible":false,
"opacity":1.00,
"url":"http://maps.raleighnc.gov/ArcGIS/rest/services/BaseMapBasic/MapServer"
}
],
"imagelayers":
[
{
 "id":"image2012",
 "label":"2012",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos2012/ImageServer",
 "countywide":false
},
{
 "id":"image2011",
 "label":"2011",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos2011/ImageServer",
 "countywide":false
},
{
 "id":"image2010",
 "label":"2010",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos2010/ImageServer",
 "countywide":true
},
{
 "id":"image2009",
 "label":"2009",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos2009/ImageServer",
 "countywide":false
},
{
 "id":"image2008",
 "label":"2008",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos2008/ImageServer",
 "countywide":false
},
{
 "id":"image2007",
 "label":"2007",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos2007/ImageServer",
 "countywide":false
},
{
 "id":"image2006",
 "label":"2006",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos2006/ImageServer",
 "countywide":false
},
{
 "id":"image2005",
 "label":"2005",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos2005/ImageServer",
 "countywide":true
},
{
 "id":"image2004",
 "label":"2004",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos2004/ImageServer",
 "countywide":false
},
{
 "id":"image2002",
 "label":"2002",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos2002/ImageServer",
 "countywide":false
},
{
 "id":"image1999",
 "label":"1999",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos1999/ImageServer",
 "countywide":true
},
{
 "id":"image1988",
 "label":"1988",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos1988/ImageServer",
 "countywide":true
},
{
 "id":"image1981",
 "label":"1981",
 "type":"tiled",
 "visible":false,
 "opacity":1.00,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Orthos10/Orthos1981/ImageServer",
 "countywide":true
}
],"oplayers":[
 {"id":"corporate",
 "label":"Corporate Boundaries",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Planning/Jurisdictions/MapServer",
 "opacity":0.70,
 "minscale":19200,
 "inlist":true},
 {"id":"publicsafety",
 "label":"Public Safety",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/PublicSafety/PublicSafety/MapServer",
 "opacity":0.60,
 "inlist":true}, 
 {"id":"zoning",
 "label":"Zoning",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Planning/Zoning/MapServer",
 "opacity":0.50,
 "visiblelayers":[-1],
 "minscale":76800,
 "inlist":true},  
 {"id":"leafcollection",
 "label":"Leaf Collection Areas",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/PublicWorks/LeafCollection/MapServer",
 "opacity":0.50,
 "inlist":true},
 {"id":"floodplains",
 "label":"Floodplains",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Environmental/Floodplains/MapServer",
 "opacity":0.60,
  "minscale":76800,
 "inlist":true,
 "template":{
  "fieldInfos":[
    {"fieldName":"ZONE_IMAPS",
    "label":"Zone"}
  ]
 }},    
 {"id":"watersheds",
 "label":"Watersheds",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Environmental/Watersheds/MapServer",
 "opacity":0.50,
 "inlist":true},  
 {"id":"swiftcreek",
 "label":"Swift Creek Critical Area",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Environmental/SwiftCreek/MapServer",
 "opacity":0.75,
 "inlist":true},  
 {"id":"riparianbuffers",
 "label":"Riparian Buffers",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Environmental/RiparianBuffers/MapServer",
 "opacity":0.50,
 "minscale":19200,
 "inlist":true},  
 {"id":"parcels",
 "label":"Parcels",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Parcels/MapServer",
 "opacity":1.00,
 "minscale":9600,
 "inlist":true}, 
 {"id":"topography",
 "label":"Topography",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Environmental/Topo/MapServer",
 "opacity":1.00,
 "minscale":4800,
 "inlist":true}, 
 {"id":"greenways",
 "label":"Greenways",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Parks/Greenway/MapServer",
 "opacity":1.00,
 "inlist":true}, 
 {"id":"bluelinestreams",
 "label":"USGS Blueline Streams",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Environmental/USGS_Streams/MapServer",
 "opacity":1.00,
 "inlist":true},    
 {"id":"easements",
 "label":"Easements",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/PublicUtility/Easements/MapServer",
 "opacity":1.00,
 "minscale":19200,
 "inlist":true}, 
 {"id":"sewer",
 "label":"Sewer",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/PublicUtility/SewerExternal/MapServer",
 "opacity":1.00,
 "minscale":19200,
 "inlist":true},  
 {"id":"stormwater",
 "label":"Raleigh Stormwater",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/PublicWorks/Stormwater/MapServer",
 "opacity":1.00,
 "inlist":true},  
 {"id":"devplans",
 "label":"Raleigh Development Plans",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Planning/RalDevPlans/MapServer",
 "opacity":1.00,
 "minscale":19200,
 "inlist":true}, 
 {"id":"overlays",
 "label":"Zoning Overlays",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Planning/Overlays/MapServer",
 "opacity":1.00,
 "minscale":76800,
 "inlist":true}, 
 {"id":"subdivisions",
 "label":"Subdivisions",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Planning/Subdivisions/MapServer",
 "opacity":1.00,
 "minscale":19200,
 "inlist":true},    
 {"id":"septic",
 "label":"Septic",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Environmental/SepticTanks/MapServer",
 "opacity":1.00,
 "minscale":38400,
 "inlist":true}, 
 {"id":"wells",
 "label":"Wells",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Environmental/Wells/MapServer",
 "opacity":1.00,
 "minscale":19200,
 "inlist":true},
 {"id":"census",
 "label":"Demographics",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Census/MapServer",
 "opacity":1.00,
 "inlist":true},   
 {"id":"specialuse",
 "label":"Special Zoning Uses",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Planning/SpecialUse/MapServer",
 "opacity":0.60,
 "inlist":true},
 {"id":"boundaries",
 "label":"Boundaries",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Boundaries/MapServer",
 "opacity":0.80,
 "inlist":true},  
 {"id":"urbanprogress",
 "label":"Raleigh Urban Progress Zone",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Planning/UrbanProgressZone/MapServer",
 "opacity":0.50,
 "minscale":76800,
 "inlist":true},                        
 {"id":"urbanform",
 "label":"Raleigh Urban Form",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Planning/UrbanForm/MapServer",
 "opacity":0.80,
 "minscale":100000,
 "inlist":true},
 {"id":"futurelanduse",
 "label":"Raleigh Future Land Use",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Planning/FutureLandUse/MapServer",
 "opacity":0.60,
 "inlist":true},
 {"id":"redevelopment",
 "label":"Raleigh Redevelopment Areas",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Planning/RedevelopmentAreas/MapServer",
 "opacity":1.00,
 "minscale":76800,
 "inlist":true},
{"id":"permits",
 "label":"Raleigh Permit History",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Planning/Permit_History/MapServer",
 "opacity":1.00,
 "minscale":38400,
 "inlist":true},
{"id":"poi",
 "label":"Places of Interest",
 "type":"dynamic",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/POI1/MapServer",
 "opacity":1.00,
 "minscale":76800,
 "inlist":true} 
],"labels": {"id":"labels",
 "label":"Labels",
 "type":"tiled",
 "visible":false,
 "url":"http://maps.raleighnc.gov/ArcGIS/rest/services/Labels/MapServer",
 "opacity":1.00,
 "inlist":false}
},
"tools":[
 {
  "id":"identify",
  "label":"Identify",
  "icon":"./img/info_default.png",
  "toggle":true
 },
 {  
  "id":"select",
  "label":"Select",
  "icon":"./img/select_point_default.png",
  "toggle":true
 },
 {
  "id":"streetview",  
  "label":"Streetview",
  "icon":"./img/streetview_default.png",
  "toggle":true
 },
 {
  "id":"pictometry",  
  "label":"Pictometry",
  "icon":"./img/pictometry_default.png",  
  "toggle":true
 },
 {
  "id":"measure",  
  "label":"Measure",
  "icon":"./img/measure_default.png",
  "toggle":false
 },
 {
  "id":"historic",
  "label":"Historic",
  "icon":"./img/clock_default.png",
  "toggle":false
 },
 {
  "id":"bookmarks",  
  "label":"Bookmarks",
  "toggle":false,
  "icon":"./img/bookmark_default.png",
  "script":"js/plugins/jquery.bookmarks.js"
 },
 {
  "id":"markup",    
  "icon":"./img/markup_default.png",
  "label":"Markup",
  "toggle":false
 },
 {
  "id":"print",  
  "label":"Print",
  "icon":"./img/print_default.png",
  "script":"js/plugins/jquery.print-dialog.js",  
  "toggle":false,
  "parameters":{
    "layouts":["A3 Landscape",
                "A3 Portrait",
                "A4 Landscape",
                "A4 Portrait",
                "Letter ANSI A Landscape",
                "Letter ANSI A Portrait",
                "Tabloid ANSI B Landscape",
                "Tabloid ANSI B Portrait",
                "MAP_ONLY"],
    "formats":[
                "PDF",
                "PNG32",
                "PNG8",
                "JPG",
                "GIF",
                "EPS",
                "SVG",
                "SVGZ"
            ]
  }
 },
 {
  "id":"clear",  
  "label":"Clear",
  "icon":"./img/clear_default.png",  
  "toggle":false
 }],
"panel":
{
 "tools":[
  {
  	"label":"Property Search",
    "divid":"propertydiv",
    "script":"js/plugins/jquery.property-search.js"
  },
  {
  	"label":"Location Search",
    "divid":"locationdiv",
    "script":"js/plugins/jquery.location-search.js"
  },
  {
  	"label":"Layers",
    "divid":"layersdiv",
    "script":"js/plugins/jquery.layer-list.js"    
  }
 ]
},
"property":
{
  "soe":"http://mapstest.raleighnc.gov/arcgis/rest/services/Parcels/MapServer/exts/PropertySOE",
  "realEstateOp":"RealEstateSearch",
  "photoOp":"PhotoSearch",
  "deedOp":"DeedSearch",
  "addressOp":"AddressSearch",
  "septicOp":"Septic Permits",
  "propertyService":"http://mapstest.raleighnc.gov/arcgis/rest/services/Parcels/MapServer/0",
  "symbolMultiple":{
    "type":"esriSFS",
    "style":"esriSFSNull",
    "color":[255,0,0],
    "outline":{
      "type":"esriSLS",
      "style":"esriSLSSolid",
      "color":[255,255,0],
      "width":2
    }
  },
  "symbolSingle":{
    "type":"esriSFS",
    "style":"esriSFSNull",
    "color":[255,0,0],
    "outline":{
      "type":"esriSLS",
      "style":"esriSLSSolid",
      "color":[255,0,0],
      "width":2
    }
  }
},"location":
{
  "geocoder":"http://maps.raleighnc.gov/arcgis/rest/services/Locators/WakeLocator/GeocodeServer",
  "intersections":"http://maps.raleighnc.gov/arcgis/rest/services/StreetsDissolved/MapServer/0",
  "poi":"http://maps.raleighnc.gov/arcgis/rest/services/POI1/MapServer/0"
},
  "bookmarks":[
    {"name":"Downtown Raleigh", "extent":
      {
    "xmin":2104769,
    "ymin":736771,
    "xmax":2109252,
    "ymax":739662,
    "spatialReference":{"wkid":2264}
    }},
    {"name":"Five Points", "extent":
      {
    "xmin":2103840,
    "ymin":746959,
    "xmax":2106084,
    "ymax":748405,
    "spatialReference":{"wkid":2264}
    }}    
  ],

  "menu":{"items":[
      {"title":"About", "items":[
        {"title":"Disclaimer"},
        {"title":"About"}
      ]},
      {"title":"Feedback", "items":[
        {"title":"Feedback"},
        {"title":"Survey"}
      ]},
      {"title":"Links", "items":[
        {"title":"City of Raleigh", "content":"<a href='http://www.raleighnc.gov/' target='_blank/>"},
        {"title":"Wake County", "content":"<a href='http://www.raleighnc.gov/' target='_blank/>"}
      ]},
      {"title":"Help", "items":[
        {"title":"Help"},
        {"title":"Quickstart Guide"}
      ]}            
    ]}
};